### PR TITLE
financial-report-2023

### PR DESCRIPTION
**Pull request by Rockhung**
## Types of changes

* new post
* post update
* new draft
* draft update
* post metadata
* fix typo
* theme
* static page
* config
* dependency
* github-actions

### CODE REVIEW

**Comment by BreezeWhite** [comment]
Seems the like the error occurs because of the wrong format of the commit message. The commit message need to starts with one of the following options `new post|post update|new draft|draft update|post metadata|fix typo|theme|static page|config|dependency|github-actions`. And in this PR, it should be `new post: <the commit message here>`.

Also the content itself requires to follow a certain format as well. You can see the [financial post](https://github.com/pycontw/pycontw-blog/blob/main/content/posts/2023-02-20-financial-report.md?plain=1) last year as reference.

cc @Lee-W

File: content/.DS_Store
Comment (BreezeWhite) [inline content/.DS_Store:1]: Accidentally added this file?

**Review by BreezeWhite — COMMENTED** [review]
Sorry for the misleading about the commit message 😂 The angle bracket "<>" is not necessary. And maybe add a new commit for the CICD to run correctly, not force pushing to the old one.

**Comment by Lee-W** [comment]
@BreezeWhite Just invited you to join as a writer for this repository. Could you please coordinate with @Rockhung and see how we can get this PR merged? Thank you!

**Comment by BreezeWhite** [comment]
Filed a new PR to fix the old commit message format #86

**Pull request closed without merging**